src/main/java/bhs/devilbotz/commands/auto/AutonomousContainer.java

```diff
@@ -18,18 +18,19 @@ public class AutonomousContainer {
   private final DriveTrain drivetrain;
   private final Arm arm;
   private final Gripper gripper;
+  private final Arduino arduino;
 
-  public AutonomousContainer(DriveTrain drivetrain, Arm arm, Gripper gripper) {
+  public AutonomousContainer(DriveTrain drivetrain, Arm arm, Gripper gripper, Arduino arduino) {
     this.drivetrain = drivetrain;
     this.arm = arm;
     this.gripper = gripper;
+    this.arduino = arduino;
   }
 
   public Command getAutonomousCommand(AutonomousModes autoMode) {
     SequentialCommandGroup autonomousCommand = new SequentialCommandGroup();
     double delay = ShuffleboardManager.autoDelay.getDouble(0);
-    double maxDistance =
-        ShuffleboardManager.autoDistance.getDouble(Constants.DEFAULT_DISTANCE_MOBILITY);
+    double maxDistance = ShuffleboardManager.autoDistance.getDouble(2);
```

Comment:
```suggestion
    double maxDistance = ShuffleboardManager.autoDistance.getDouble(Constants.DEFAULT_DISTANCE_MOBILITY);
```
Hard coded to test to see if distance was greyed out because of this, can be put back